org.horizon_eda.HorizonEDA.json

```diff
@@ -0,0 +1,255 @@
+{
+    "app-id": "org.horizon_eda.HorizonEDA",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "horizon-eda",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
```

Comment:
you shouldn't duplicate x11 and fallback-x11 sockets. One or the other.

Reply:
fixed in 4d3444fb5ed94948bf6196642b02c7b02260f6c3